C++/C++.sublime-syntax

```diff
@@ -431,7 +434,7 @@ contexts:
           pop: true
         - include: expressions
 
-  namespace:
+  using-namespace:
     - match: '\b(using)\s+(namespace)\s+(?={{path_lookahead}})'
```

Comment:
We need to add handling for the instance when the user has typed just `using namespace`. Currently the `using` and `namespace` end up getting highlighted with `namespace` is a type definition. That shouldn't ever be the case.

On a high level, the issue is that it is preferable for fully-typed keywords to not shift around after a following token has been typed. Obviously this isn't possible in all cases, but we'd like to try and keep the scope changes minimal as the user types.